src/Lime.Transport.WebSocket/LimeWebSocketActivitySource.cs

```diff
@@ -0,0 +1,20 @@
+using System.Diagnostics;
+
+namespace Lime.Transport.WebSocket
+{
+    /// <summary>
+    /// Provides the ActivitySource for the Lime.Protocol WebSocket transport
+    /// </summary>
+    public static class LimeWebSocketActivitySource
+    {
+        /// <summary>
+        /// The name of the ActivitySource for the Lime.Protocol WebSocket transport
+        /// </summary>
+        public const string LimeWebSocket = "Lime.Protocol.WebSocket";
```

Comment:
I think we could match this string with the project name (which is also the Nuget package name): `Lime.Transport.Websocket`